static/data/community.ts

```diff
@@ -91,15 +91,15 @@ const communityMeetings = {
       ],
       subCards: [
         {
-          date: '1635814871',
+          date: 1635814871000,
           icon: 'film',
           buttons: [
             { text: 'Watch recordings', path: MEETING_URL },
             { text: 'Meeting Minutes', path: 'https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both' },
           ],
         },
         {
-          date: '1633395671',
+          date: 1633395671000,
```

Comment:
Would it be possible to use a more human-friendly date format and have it be converted when it's read in / consumed from here?